README.md

```diff
@@ -16,3 +16,32 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building these packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the SDK from https://downloads.creatordev.io/pistachio/marduk/OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz.
+2. Extract the SDK and go to the extracted folder.
+3. Check the feeds.conf.default whether its having an entry to this https://github.com/CreatorKit/openwrt-ckt-feeds.git
+4. Update and install the openwrt-ckt-feeds by 
+
+          ./scripts/feeds update ckt
+          ./scripts/feeds install ckt
+5. You can build a specific package from the list above by
+
+          make package/xyz/compile
+6. Once its built successfully, the generated .ipk files are placed in the bin/pistachio directory.
+7. You can either scp that ipk file to the Ci40 board running https://github.com/Creatordev/openwrt image or copy into USB stick and connect it to the board.
```

Comment:
=> To copy the .ipk files on to the Ci40 board which is running the CreatorDev/openwrt image, you can either scp them or first copy them on a USB stick, connect the stick to the board, mount it and copy the files in /tmp folder.